crates/cashu/src/nuts/nut17.rs

```diff
@@ -0,0 +1,73 @@
+//! Mint via Bitcoin Onchain
+// https://github.com/cashubtc/nuts/blob/main/17.md
```

Comment:
link leads to 404

Reply:
The nut hasn't been merged yet its still a PR. Will leave this open as a reminder to double check once it is merged but should work then